src/Uno.UI.sln

```diff
@@ -2713,6 +2715,102 @@ Global
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|ARM.Build.0 = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|iPhone.ActiveCfg = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|iPhone.Build.0 = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|iPhoneSimulator.ActiveCfg = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|iPhoneSimulator.Build.0 = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|x64.Build.0 = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Debug|x86.Build.0 = Debug|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|Any CPU.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|Any CPU.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|ARM.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|ARM.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|iPhone.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|iPhone.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|iPhoneSimulator.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|iPhoneSimulator.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|x64.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|x64.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|x86.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_Android|x86.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|Any CPU.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|Any CPU.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|ARM.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|ARM.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|iPhone.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|iPhone.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|iPhoneSimulator.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|iPhoneSimulator.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|x64.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|x64.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|x86.ActiveCfg = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_iOS|x86.Build.0 = Release|Any CPU
+		{22CF09C6-4281-49A2-A13F-477C8D539C0B}.Release_NoSamples|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
should we remove some configurations? Release_NoSamples for example

Reply:
They'll be back ultimately.